src/wp-includes/js/media/controllers/gallery-add.js

```diff
@@ -71,15 +82,19 @@ GalleryAdd = Library.extend(/** @lends wp.media.controller.GalleryAdd.prototype
 			library.unobserve( this.editLibrary );
 		}
 
-		// Accepts attachments that exist in the original library and
-		// that do not exist in gallery's library.
+		/*
+		Accept attachments that exist in the original library and
```

Comment:
Super minor: 
Please add `*` before each line in this comment like so
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#multi-line-comments

Also applies to comment below